crates/picker/src/picker.rs

```diff
@@ -381,6 +381,7 @@ impl<D: PickerDelegate> Render for Picker<D> {
         div()
             .key_context("Picker")
             .size_full()
+            .cursor_pointer()
```

Comment:
Why do we need that?
This PR's changes are about recent project menus, and you've added the proper hand cursor there in https://github.com/zed-industries/zed/pull/8595 already.

Also, this breaks the footer caret: now it's a hand always, but the footer might not be clickable.

Reply:
I revert this change.